docs/ReleaseNotes.md

```diff
@@ -23,6 +23,7 @@ The included licenses apply to the following files:
 Place release notes for the upcoming release below this line and remove this line upon naming this release.
 
 - The incomplete WaveMatrix implementation has been removed.
+- The DXIL validator supports the string table and semantic index table in any order.
```

Comment:
It feels like this relnote needs a bit more context.  How about:
> DXIL container validation for `PSV0` part allows any content ordering inside string and semantic index tables.

Reply:
Updated.